ocaml/xapi/quicktest_cbt.ml

```diff
@@ -23,7 +23,9 @@ let start session_id =
                    (ExnHelper.string_of_exn error)) in
 
   let test_assert ~test op ~msg =
-    try assert op with assert_failure -> failed test msg in
+    try assert op with (Assert_failure _) ->
+      debug test msg;
+      raise (Failure "test assertion failed") in
```

Comment:
Another way would be to declare our own exception here, and throw that:
```
exception Test_assertion_failed
...
raise Test_assertion_failed
```
This is nicer in my opinion, but there is no need to change this though.